contrib/test/integration/golang.yml

```diff
@@ -1,9 +1,14 @@
 ---
 
+- name: ensure Golang dir is empty first
+  file:
+    path: /usr/local/go
+    state: absent
+
 - name: fetch Golang
   unarchive:
     remote_src: yes
-    src: https://storage.googleapis.com/golang/go1.8.4.linux-amd64.tar.gz
+    src: https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz
```

Comment:
Is this for using 1.9.x just with master?

Reply:
Nope, I think this is using it globally, do we have any issue in using it for other branches as well?